.github/workflows/asv-regular.yml

```
@@ -0,0 +1,58 @@
+name: asv-benchmarks-daily
+
+on:
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  asv-run:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout sunpy repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+
+      - name: Checkout sunpy-benchmarks repo
+        uses: actions/checkout@v2
+        with:
+          repository: sunpy/sunpy-benchmarks
+          ref : main
+          path: asv_results
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+
+      - name: Install dependencies
+        run : |
+          python -m pip install --upgrade pip
+          pip install asv virtualenv
+
+      - name: Add machine info for ASV
+        run : asv machine --machine GH-Actions --os ubuntu-20.04 --arch x64 --cpu "2-core unknown" --ram 7GB
+
+      - name: Run benchmarks for commits since v2.1
+        run : taskset -c 0 asv run --skip-existing-successful 2.1..

```

```suggestion
        run : taskset -c 0 asv run --skip-existing-successful v2.1.dev..
```

The v2.1.dev tag is on master at the point juat after we released 2.0 so that's probably the best place to start.

---

I don't remember whether I've tried this. Not sure. But if you want, I'll change this tag and we'll see tomorrow the result